scripts/common/preflights.sh

```diff
@@ -638,3 +639,17 @@ allow_remove_docker_new_install() {
           logWarn "conflicts, please uninstall Docker and re-run the install script."
      fi
 }
+
+allow_migrate_from_rook_to_openebs() {
+    if [ -z "$ROOK_VERSION" ] && [ -n "$OPENEBS_VERSION" ]; then
+        if commandExists kubectl; then
+            if kubectl get ns | grep -q rook-ceph; then
+                if [ $(echo "$OPENEBS_VERSION" | cut -d. -f1) == 2 ] || [ $(echo "$OPENEBS_VERSION" | cut -d. -f1) == 1 ] || [ "$OPENEBS_VERSION" == "3.2.0" ]; then
```

Comment:
could you use the built in semverParse function? this seems more readable

```suggestion
                semverParse "$OPENEBS_VERSION"
                # if $OPENEBS_VERSION is less than 3.3.0
                if [ "$major" -lt "3" ] || { [ "$major" = "3" ] && [ "$minor" -lt "3" ] ; }; then
```

Reply:
another option is semverCompare

```suggestion
                semverCompare "$OPENEBS_VERSION" "3.3.0"
                # if $OPENEBS_VERSION is less than 3.3.0
                if [ "$SEMVER_COMPARE_RESULT" = "-1" ]; then
```